\documentclass[a4paper]{article}
\usepackage{placeins}
\usepackage{graphics}
\usepackage[dvipsnames]{xcolor}
\usepackage{natbib}
\usepackage{graphicx}
\usepackage{placeins}
\usepackage{float}
\usepackage{url}
\usepackage{subfig}
\usepackage{hyperref}

\hypersetup{
  colorlinks=true,
  linkcolor=Blue,
  citecolor=Blue,
  filecolor=magenta,      
  urlcolor=cyan,
 }


\begin{document}

\title{Supplementary : data analysis methods}

\tableofcontents

\section{Cell type annotation}

\subsection{Scatter plots}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/label.png}
  \caption{Scatter: Time points (days) and sorted assays (CD) annotation}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/leiden_identity_simple2.png}
  \caption{Scatter: Leiden identity simple annotation}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/leiden_identity.png}
  \caption{Scatter: Leiden identity annotation}
\end{figure}

\FloatBarrier
\subsection{Bar plots}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/label_proportions.png}
  \caption{Proportions: Leiden identity simple annotation}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/leiden_identity_simple2_proportions.png}
  \caption{Proportions: Leiden identity simple annotation}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/leiden_identity_proportions.png}
  \caption{Proportions: Leiden identity annotation}
\end{figure}

\FloatBarrier
\subsection{Correlation plots}

\begin{figure}[!htb]
  \centering
  \subfloat[RNA CD]{
    \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/correlation_matrix_leiden_identity_simple2_adata_RNA_CD.png}
  }
  \subfloat[RNA days]{
    \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/correlation_matrix_leiden_identity_simple2_adata_RNA_days.png}
  }
  \hspace{0mm}
  \subfloat[ATAC CD]{
    \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/correlation_matrix_leiden_identity_simple2_adata_ATAC_CD.png}
  }
  \subfloat[ATAC days]{
    \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/correlation_matrix_leiden_identity_simple2_adata_ATAC_days.png}
  }
  \caption{Correlation between cell type averages (obtained using top PCs)}
\end{figure}

\FloatBarrier
\section{Data integration}

\subsection{Linear optimal transport}

\subsubsection{Scatter plots}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_RNA_days_subspace.png}
  \caption{Assays in RNA days PCA subspace}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_RNA_days_subspace_aligned.png}
  \caption{Assays in RNA days PCA subspace, after alignment with linear optimal transport}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_RNA_days_subspace_aligned_Intersection_of_expressed_genes_leiden_identity_simple2.png}
  \caption{Assays in RNA days PCA subspace obtained using the intersection of expressed genes, after alignment with linear optimal transport.}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_RNA_days_subspace_aligned_Union_of_variable_genes_leiden_identity_simple2.png}
  \caption{Assays in RNA days PCA subspace obtained using the union of variable genes, after alignment with linear optimal transport.}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_RNA_days_subspace_aligned_Intersection_of_variable_genes_leiden_identity_simple2.png}
  \caption{Assays in RNA days PCA subspace obtained using the intersection of variable genes, after alignment with linear optimal transport.}
\end{figure}

\FloatBarrier
\subsubsection{Correlation plots}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_X_pca_target_leiden_identity_simple2.png}
  \caption{Correlation between assays' cell type averages (obtained using top PCs of assays, each in their own PCA subspace)}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_X_pca_target_leiden_identity_simple2.png}
  \caption{Correlation between assays' cell type averages (obtained using top PCs of assays in RNA days PCA subspace)}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=\textwidth]{../figures/hematopoiesis/assays_X_pca_transp_leiden_identity_simple2.png}
  \caption{Correlation between assays' cell type averages (obtained using top PCs of assays in RNA days PCA subspace, after alignment with linear optimal transport)}
\end{figure}

\FloatBarrier
\subsection{Supervised optimal transport}

\textbf{Grouping of categories to go from full to simple annotation}
\begin{itemize}
  \item 'Early-ERP','Erythroblast','CD34+ ERP' = 'Erythroid'
  \item 'Platelet','CD34+ MKP' = 'MKP'
  \item 'Pre-Dendritic','Dendritic Cell' = 'Dendritic'
  \item 'CD34+ CLP','CD34+ pre-B','Pro-B','Plasma Cell', 'NK cells','Naive T-cell','CD8 T-cell' = 'Lymphoid'
  \item 'CD34+ Mixed-Lineage', 'CD34+ HSC','CD34+ CMP', 'CD34+ Gran', 'Eosinophil','Stromal Cells' = 'Mixed-Lineage'
  \item change unlikely 'Erythroid' annotation (outlier, mixed cluster) for late ATAC days (day 7 and 12) to 'Mixed-Lineage'
\end{itemize}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/integration_supervision.png}
  \caption{Supplementary: supervision of OT cost matrix}
\end{figure}


\FloatBarrier
\section{HSC score}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/integrated_hsc_score.png}
  \caption{Supplementary: hematopoeitic stem cell score}
\end{figure}






\FloatBarrier
\section{Trajectory inference}

\subsection{Trajectory in PCA space}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/PC_pairwise_trajectory.png}
  \caption{Supplementary: Trajectory visualized in PCA space}
\end{figure}


\FloatBarrier
\section{Trajectory analysis}
\subsection{Erythroid branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_single_branch_pseudotime.png}
  \caption{Supplementary: Erythroid branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Erythroid_40_103_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}


\subsection{MKP branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_single_branch_pseudotime.png}
  \caption{Supplementary: MKP branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/MKP_40_110_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}


\FloatBarrier
\subsection{Basophil branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_single_branch_pseudotime.png}
  \caption{Supplementary: MKP branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Basophil_40_109_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}



\FloatBarrier
\subsection{Neutrophil branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_single_branch_pseudotime.png}
  \caption{Supplementary: MKP branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Neutrophil_40_106_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}


\FloatBarrier
\subsection{Monocyte branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_single_branch_pseudotime.png}
  \caption{Supplementary: MKP branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Monocyte_40_15_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}



\FloatBarrier
\subsection{Myeloid DC branch}

\subsubsection{Pseudotime}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_single_branch_pseudotime.png}
  \caption{Supplementary: MKP branch pseudotime}
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_none_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_magic_single_branch_volcanos_motifs.png}
  \caption{Supplementary: Erythroid branch marker detection with Spearman correlation and MAGIC imputed matrices for gene score and RNA expression. The volcano plot is colored by -log10 adjusted p-values}
\end{figure}

\FloatBarrier
\subsubsection{Heatmap}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_single_smooth_none_heatmap_grouped_assays.png}
  \caption{Supplementary: Heatmap of top negative and positive correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}

\FloatBarrier
\subsubsection{X plots}
 
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_none_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_none_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_none_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays}
\end{figure}


\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_magic_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, for all pairs of assays, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_magic_semi_detailed_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping sorted and timepoints assays for RNA and gene score, with MAGIC smoothing for gene score and RNA expression}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/Myeloid DC_40_107_smooth_magic_grouped_X_plot.png}
  \caption{Supplementary: X-plot showing Spearman correlations against pseudotime, grouping all RNA and chromVAR assays with MAGIC smoothing for gene score and RNA expression}
\end{figure}


\FloatBarrier
\section{Early populations}

\subsection{Top markers from lineage tracing}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/early_groups_4_genes_panel.png}
  \caption{Supplementary: Marker genes previously identified by lineage tracing distinguish early monocyte populations: dendritic-like and neutrophil-like}
\end{figure}

\FloatBarrier
\subsection{Early populations classification by backpropagating labels}

\subsubsection{UMAP}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/early_groups_umap.png}
  \caption{Supplementary: UMAP showing inferred dendritic-like and neutrophil-like early monocyte populations }
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/early_groups_umap_split.png}
  \caption{Supplementary: UMAP showing inferred dendritic-like and neutrophil-like early monocyte populations, split into "early" and "late" halves of pseudotime. }
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/early_groups_detailed_assays.png}
  \caption{Supplementary: Volcano plots of Wilcoxon test between inferred dendritic-like and neutrophil-like early monocyte populations, for "early" and "late" halves of pseudotime. Motifs and RNA expression show TFs only while RNA expression all shows all genes}
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/early_groups_grouped_assays.png}
  \caption{Supplementary: Volcano plots of Wilcoxon test between inferred dendritic-like and neutrophil-like early monocyte populations, for "early" and "late" halves of pseudotime, grouping assays. Motifs and RNA expression show TFs only while RNA expression all shows all genes}
\end{figure}

\FloatBarrier
\subsection{Early populations classification using known lineage tracing markers}

\subsubsection{UMAP}
\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/markers_early_groups_umap.png}
  \caption{Supplementary: UMAP showing inferred dendritic-like and neutrophil-like early monocyte populations }
\end{figure}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/markers_early_groups_umap_split.png}
  \caption{Supplementary: UMAP showing inferred dendritic-like and neutrophil-like early monocyte populations, split into "early" and "late" halves of pseudotime. }
\end{figure}

\FloatBarrier
\subsubsection{Volcanos}

\begin{figure}[!htb]
  \centering
  \includegraphics[width=0.5\textwidth]{../figures/hematopoiesis/markers_early_groups_detailed_assays.png}
  \caption{Supplementary: Volcano plots of Wilcoxon test between inferred dendritic-like and neutrophil-like early monocyte populations, for "early" and "late" halves of pseudotime. Motifs and RNA expression show TFs only while RNA expression all shows all genes}
\end{figure}


\end{document}